spark/src/test/scala/ai/chronon/spark/test/TableUtilsTest.scala

```diff
@@ -547,4 +554,75 @@ class TableUtilsTest {
       assertTrue(firstDs.contains("2022-11-01"))
     }
   }
+
+  @Test
+  def testGetPartitionsWithLongPartition(): Unit = {
+    val tableName = "db.test_long_partitions"
+    spark.sql("CREATE DATABASE IF NOT EXISTS db")
+    val structFields = Array(
+      StructField("dateint", LongType),
+      StructField("hour", IntType),
+      StructField("event_type", StringType),
+      StructField("label_ds", StringType),
+      StructField("feature_value", IntType)
```

Comment:
Let's use camel case for consistency. 
```suggestion
    val structFields = Array(
      StructField("dateInt", LongType),
      StructField("hour", IntType),
      StructField("eventType", StringType),
      StructField("labelDs", StringType),
      StructField("featureValue", IntType)
```